usmqe_tests/api/tendrl/README.rst

```diff
@@ -0,0 +1,3 @@
+There are 2 types of tests here:
```

Comment:
I am a little afraid of this nesting, but probably it's the best what we can do. 
BTW I missed where should go log-in tests, and maybe some others too? They are not gluster nor ceph. They could be in tendrl directory itself or we could have common directory for them similar to api. Anyway please change the readme appropriately.